backend/hitas/views/utils/serializers.py

```diff
@@ -32,12 +32,12 @@ def get_id(self, instance: ExternalSafeDeleteHitasModel) -> str:
     def to_internal_value(self, data):
         """If referenced in another serializer by id, return an object"""
         if not self.instance and isinstance(data, dict):
-            id = data.get("id", None)
-            if id is not None:
+            id_ = data.get("id", None)
+            if id_ is not None:
```

Comment:
Perfect place to use the Walrus operator 😉 

`if (id_ := data.get("id", None)) is not None:`

Reply:
I wonder if SonarCloud will think the Walrus increases complexity too much 😅 It already complained once about using it in a different context, but I'll see what it thinks now.

Reply:
SonarLint says it's fine, so in it goes!